tests/kani/Unwind-Attribute/fixme_lib.rs

```diff
@@ -0,0 +1,38 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+// rmc-flags: --no-unwinding-checks
+
+// Fix me
```

Comment:
Please add the reason as to why this is a fix me test, including relevant issues if possible.